scripts/Muon/GUI/Common/corrections_tab_widget/corrections_view.py

```diff
@@ -0,0 +1,136 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantidqt.utils.observer_pattern import GenericObserver
+from mantidqt.utils.qt import load_ui
+
+from Muon.GUI.Common.corrections_tab_widget.dead_time_corrections_view import DeadTimeCorrectionsView
+from Muon.GUI.Common.fitting_widgets.basic_fitting.workspace_selector_view import WorkspaceSelectorView
+from Muon.GUI.Common.message_box import warning
+
+from qtpy.QtWidgets import QWidget
+
+ui_form, widget = load_ui(__file__, "corrections_view.ui")
+
+
+class CorrectionsView(widget, ui_form):
+    """
+    The CorrectionsView contains widgets allowing a Dead Time correction and Background correction.
+    """
+
+    def __init__(self, parent: QWidget = None):
+        """Initializes the CorrectionsView."""
+        super(CorrectionsView, self).__init__(parent)
+        self.setupUi(self)
+
+        self.run_selector = WorkspaceSelectorView(self)
+        self.run_selector.set_workspace_combo_box_label("Runs :")
+        self.run_selector.workspace_combo_box_label.setMinimumWidth(50)
+        self.run_selector.workspace_combo_box_label.setMaximumWidth(50)
+        self.run_selector_layout.addWidget(self.run_selector)
+
+        self.dead_time_corrections_view = DeadTimeCorrectionsView(self)
+        self.dead_time_layout.addWidget(self.dead_time_corrections_view)
+
+        self.disable_tab_observer = GenericObserver(self.disable_view)
+        self.enable_tab_observer = GenericObserver(self.enable_view)
+
+        self.disable_view()
+
+    def set_slot_for_run_selector_changed(self, slot) -> None:
```

Comment:
do we need all of these here? Is there a way to just pass the run selector information into the dead time widget? Im just concerned that when we add the other corrections this file will become huge

Reply:
These need to be here if we want to connect to the slots in the presenter. It also follows the same methodology used for the fitting widgets

Reply:
Is there a way to make it shorter?

We are going to add code for background corrections and high order dead time corrections

Reply:
I have an idea to make it shorter

Reply:
I've separated the dead time stuff from the rest, so we can have the background corrections in separate MVP files. So the files are now shorter, and dead time and background corrections will be decoupled